sds_data_manager/lambda_images/instruments/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# TODO I think we should have a single Dockerfile and pyproject.toml
+# for all dockerized lambdas
+FROM public.ecr.aws/lambda/python:3.11
+
+# Install git
+RUN yum install -y git
+
+# Install development tools and required libraries
+RUN pip install psycopg2-binary
```

Comment:
This should be in the poetry project, so you either pip install the existing requirements.txt file for `lambda-dev` or we make a new dependency group and a new requirements.txt file. Let me know if you have other questions on how to do that.

Reply:
Wait, for Dockerized lambdas we actually require a Dockerfile. Or am I missing something?